### ISSUE TITLE
Bug: Types are not found wth `nodenext` module resolution

### ISSUE THREAD

**Issue by bramvbilsen** [Hexagon/croner#168]
The types for `croner` are not being found in my Typescript application. I installed the package as follows:
```
npm i croner
```
The specific errors:
```
'.../node_modules/croner/dist/croner.min.cjs' implicitly has an 'any' type.
```
I made sure to include all node_modules paths for types in my `tsconfig.json` file:
```
// tsconfig.json
{
    ...
    "moduleResolution": "nodenext",
    "typeRoots": [
        "./src/types/*",
        "./node_modules/*",
        "./node_modules/croner/types/*" // Added this for completeness sake
    ],
    ...
}
```
The issue appears to be with `moduleResolution` being `nodenext`. The import does not show any errors if I switch it to use `node`. But, unfortunately, this breaks compatibility for other packages and is not an option.

Any ideas?

**Comment by Hexagon** [comment]
Hmm, interesting! Will have a look at that later, which version of typescript do you use?

**Comment by bramvbilsen** [comment]
> Hmm, interesting! Will have a look at that later, which version of typescript do you use?

Wow, thanks for the quick reply!
Using Typescript version `^4.8.4`

**Comment by Hexagon** [comment]
Found some pointers here, https://github.com/microsoft/TypeScript/issues/50466

Will fix this asap :+1:

**Comment by Hexagon** [comment]
@bramvbilsen Trying to set up a test environment to reproduce this behavior. Do you have `"type": "module"` or not in your `package.json`? How does your import statement look like?

**Comment by bramvbilsen** [comment]
@Hexagon No the `package.json` is just the default `"type": "commonjs"`

**Comment by Hexagon** [comment]
Oh, i see. Seems like croner need to export separate type definitions for commonjs (`.d.cts`)... Interesting stuff indeed!

**Comment by Hexagon** [comment]
@bramvbilsen Proposed fix released in `6.0.0-dev.0`

Please test by

```npm install croner@dev`

**Comment by Hexagon** [comment]
Released in `6.0.0`. Please re-open if you still have problems with this.

**Comment by bramvbilsen** [comment]
My apologies for the late reply. Version `6.0.0` works perfectly, thank you!
